plugins/org.eclipse.xtext/src/org/eclipse/xtext/workspace/FileWorkspaceConfig.xtend

```diff
@@ -93,4 +98,25 @@ class FileSourceFolder implements ISourceFolder {
 		'''�name� (�path�)'''
 	}
 	
+}
+
+@Data
+class SingleProjectWorkspaceConfig implements IWorkspaceConfig {
+	
+	IProjectConfig projectConfig
+	
+	override findProjectByName(String name) {
+		if (projectConfig.name == name)
+			return projectConfig
```

Comment:
I had that before and then decided to remove them. I think as long as the methods are this short, the code reads nicely without the explicit return null.